app/scripts/controllers/newOverview.js

```diff
@@ -0,0 +1,1200 @@
+  var deploymentIsInProgress = $filter('deploymentIsInProgress');
+  var getErrorDetails = $filter('getErrorDetails');
+  var imageObjectRef = $filter('imageObjectRef');
+  var isJenkinsPipelineStrategy = $filter('isJenkinsPipelineStrategy');
+  var isNewerResource = $filter('isNewerResource');
+  var label = $filter('label');
+  var orderObjectsByDate = $filter('orderObjectsByDate');
+  var getPodTemplate = $filter('podTemplate');
+
+  var imageStreams;
+  var labelSuggestions = {};
+  var pipelineLabelSuggestions = {};
+
+  // The most recent replication controller by deployment config name. This
+  // might not be the active deployment if failed or cancelled.
+  var mostRecentByDeploymentConfig = {};
+
+  // `overview.state` tracks common state that is shared with overview-list-row.
+  // This avoids having to pass the same values as attributes again and again
+  // for different types, but lets us update these maps in one place as needed
+  // from watch callbacks in the overview controller.
+  //
+  // NOTE: Do not change or remove properties without updating overview-list-row.
+  var state = overview.state = {
+    alerts: {},
+    builds: {},
+    clusterQuotas: {},
+    imageStreamImageRefByDockerReference: {},
+    imagesByDockerReference: {},
+    limitRanges: {},
+    limitWatches: limitWatches,
+    notificationsByObjectUID: {},
+    pipelinesForDeploymentConfig: {},
+    podsByOwnerUID: {},
+    quotas: {},
+    recentPipelinesByDeploymentConfig: {},
+    routesByService: {},
+    servicesByObjectUID: {},
+    // Set to true below when metrics are available.
+    showMetrics: false
+  };
+
+  AlertMessageService.getAlerts().forEach(function(alert) {
+    state.alerts[alert.name] = alert.data;
+  });
+  AlertMessageService.clearAlerts();
+
+  var getBreakpoint = function() {
+    if (window.innerWidth < BREAKPOINTS.screenSmMin) {
+      return 'xs';
+    }
+
+    if (window.innerWidth < BREAKPOINTS.screenMdMin) {
+      return 'sm';
+    }
+
+    if (window.innerWidth < BREAKPOINTS.screenLgMin) {
+      return 'md';
+    }
+
+    return 'lg';
+  };
+
+  // Track the breakpoint ourselves so we can remove elements from the page,
+  // rather than hiding them using CSS. This avoids rendering charts more than
+  // once for the responsive layout, which switches to tabs at smaller screen
+  // widths.
+  overview.state.breakpoint = getBreakpoint();
+  var onResize = _.throttle(function() {
+    var breakpoint = getBreakpoint();
+    if (overview.state.breakpoint !== breakpoint) {
+      $scope.$evalAsync(function() {
+        overview.state.breakpoint = breakpoint;
+      });
+    }
+  }, 50);
+
+  $(window).on('resize', onResize);
+
+  overview.renderOptions = {
+    showGetStarted: false,
+    showLoading: true
+  };
+
+  overview.filterByOptions = [{
+    id: 'name',
+    label: 'Name'
+  }, {
+    id: 'label',
+    label: 'Label'
+  }];
+  overview.filterBy = 'name';
+
+  overview.viewByOptions = [{
```

Comment:
It's funny, I was debating removing `renderOptions` since some directives that don't have isolated scope look for it. For instance,

https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/nav.js#L150-L154

The settings here apply only to the overview, but `renderOptions` implies that they might be used outside of this controller. We have some copied code in other controllers as well that sets `renderOptions` when not needed.

As an aside, we might remove that check from `project-filter`. I don't think it's needed anymore since it was removed from the nav a while back?